azkaban-execserver/src/main/resources/execute-as-user.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
```

Comment:
Are we building this? If so, perhaps it shouldn't be in `resources` but rather in one of the standard directories according to [Gradle's docs on building native binaries](https://docs.gradle.org/current/userguide/nativeBinaries.html)?